src/components/MarkdownProvider/components/Code.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+import ReactSyntaxHighlighter from 'react-syntax-highlighter';
+import dracula from 'react-syntax-highlighter/dist/esm/styles/hljs/dracula';
+
+const StyledPre = styled.pre`
+  margin: ${p => p.theme.space.md} 0;
+  border-radius: ${p => p.theme.borderRadii.md};
+  /* stylelint-disable-next-line declaration-no-important */
+  padding: ${p => p.theme.space.sm} !important;
+  overflow: auto;
+`;
```

Comment:
Set font family stack to `p.theme.fonts.mono`.

Reply:
This is already set with our `css-bedrock` reset. What is the reason for us to override this locally?

Reply:
Right, forgot about that.